src/testRunner/unittests/tscWatch/emitAndErrorUpdates.ts

```diff
@@ -1,7 +1,7 @@
 namespace ts.tscWatch {
     describe("unittests:: tsc-watch:: Emit times and Error updates in builder after program changes", () => {
         const config: File = {
-            path: `${projectRoot}/tsconfig.json`,
+            path: `${tscWatch.projectRoot}/tsconfig.json`,

```

Comment:
why is this needed if we are already in the ts.tscWatch namespace?

Reply:
it's now gone~